src/Mollie/WC/Helper/IconFactory.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+use Mollie\Api\Types\PaymentMethod;
+
+class Mollie_WC_Helper_IconFactory
+{
+
+    /**
+     * @param $data
+     * @return string
+     */
+    public function create($data)
+    {
+        if (!preg_match('/[^A-Za-z0-9]/', $data)) {
```

Comment:
Sorry I do not understand this part, could you explain me why a preg-match against an alphanumeric value if that value is the ID of the payment method? Which I suppose it's a string.